strawberryfields/configuration.py

```diff
@@ -15,28 +15,136 @@
 This module contains functions used to load, store, save, and modify
 configuration options for Strawberry Fields.
 """
+import collections
 import os
 
 import toml
 from appdirs import user_config_dir
 
-from strawberryfields.logger import create_logger
 
 DEFAULT_CONFIG_SPEC = {
     "api": {
         "authentication_token": (str, ""),
         "hostname": (str, "platform.strawberryfields.ai"),
         "use_ssl": (bool, True),
         "port": (int, 443),
-    }
+    },
+    "logging": {"level": (str, "info"), "logfile": ((str, type(None)), None)},
```

Comment:
How about this one to go with the same as for `authentication_token`?
```suggestion
    "logging": {"level": (str, "info"), "logfile": (str, "")},
```